### PR TITLE
Turns the Balloon Mallet into the Hönknir

### PR DESCRIPTION
**Pull request by kaylexis**
## About The Pull Request

The balloon mallet was underwhelming, and had issues being examine while having the balloon skillchip.
TLDR: it can toss people some length, and acts as a boomerang. this is not a reference to marvels thor. i did it because i could. I didn't get a recording of the fling from the hammer because my gif recording software sucks.
clownthrow

## Why It's Good For The Game

Still entirely harmless, does a hilarious fling. Adds wearable functionality to the hammer, and gives it more sprites, back and wielded.
Unrelated note, people who get thrown by the hammer slip over peels and lubed tiles while getting flung. 😈

## Changelog

:cl:
add: makes the balloon hammer wildly hilarious
fix: fixed the weird nonexistance examine test with skillchip by removing the bit entirely
/:cl:

### CODE REVIEW

**Comment by LemonInTheDark** [comment]
should be faster

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -43,30 +43,41 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
 	max_integrity = 20
 	armor_type = /datum/armor/item_banhammer
 	resistance_flags = FIRE_PROOF
 
-/obj/item/balloon_mallet/examine(mob/user)
+/obj/item/balloon_mallet/Initialize(mapload)
 	. = ..()
-	if(HAS_TRAIT(user,TRAIT_BALLOON_SUTRA))
-		. = "A sacred weapon of the higher castes from the clown planet, used to strike fear into the hearts of their foes. Wield it with care."
+	AddElement(/datum/element/knockback, 10, FALSE, TRUE)
```
Comment (MrMelbert) [inline code/game/objects/items/weaponry.dm:68]: 1. *10* throw distance? from a balloon hammer? srs? 
2. This needs to pass the third argument (`throw_gentle`) as `TRUE` otherwise you can stunlock with it

File: code/game/objects/items/weaponry.dm [hunk 2]
```diff
@@ -43,30 +43,41 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
...
+		icon_wielded = "[base_icon_state]1", \
+		attacksound = SFX_CLOWN_STEP, \
+	)
+
+/obj/item/balloon_mallet/update_icon_state()
+	icon_state = "[base_icon_state]0"
+	return ..()
 
 /obj/item/balloon_mallet/attack(mob/living/target, mob/living/user)
+	..()
```
Comment (MrMelbert) [inline code/game/objects/items/weaponry.dm:80]: This return that you have added will break this proc (because it already calls parent later, if user is in combat mode)

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/game/objects/items/weaponry.dm (hunk 1 above)
Comment (MrMelbert) [reply code/game/objects/items/weaponry.dm]: Oh yeah I forgot about the target being the first arg my bad

**Comment by jlsnow301** [comment]
@kaylexis don't let this die pls

**Comment by jlsnow301** [comment]
> @MrMelbert pls merge..

There are merge conflicts

**Comment by jlsnow301** [comment]
> Huh

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

**Comment by jlsnow301** [comment]
next time you run into an issue like this please just ask how to fix it on [discord](https://discord.com/invite/EUvpBtU78X)

**Comment by ArcaneMusic** [comment]
Aside from merge conflicts was there any other reason it's drafted?

**Comment by LemonInTheDark** [comment]
sadge
